### PR TITLE
Upgrade pytorch version for runai

### PR DESCRIPTION
**Pull request by CorentinSeznec**
Upgrade the pytorch version in the dockerfile for the use of runai.

### CODE REVIEW

**Comment by cbovalo** [comment]
In Dockerfile, it is just a default version. The torch version you want is retrieved from `env.yaml`.

**Comment by A669015** [comment]
Please complete the title of the PR to be clear that fix the specific runai build, being backcompatible

File: Dockerfile [hunk 1]
```diff
@@ -20,8 +21,8 @@ RUN $MY_APT update && $MY_APT install -y curl gdal-bin libgdal-dev libgeos-dev g
 ENV CPLUS_INCLUDE_PATH=/usr/include/gdal
 ENV C_INCLUDE_PATH=/usr/include/gdal
 
-ARG REQUESTS_CA_BUNDLE
-ARG CURL_CA_BUNDLE
+ARG REQUESTS_CA_BUNDLE="/usr/local/share/ca-certificates/mf.crt"
```
[the remark this answers is not in the document]
Comment (CorentinSeznec) [reply Dockerfile]: I have merged main into my branch. I think that resolve the issue ?

File: Dockerfile [hunk 2]
```diff
@@ -1,10 +1,11 @@
 ARG DOCKER_REGISTRY=docker.io
-ARG TORCH_VERS=2.2.2
+ARG TORCH_VERS=2.4.1
 ARG CUDA_VERS=12.1
 
 FROM ${DOCKER_REGISTRY}/pytorch/pytorch:${TORCH_VERS}-cuda${CUDA_VERS}-cudnn9-devel
 
-ARG INJECT_MF_CERT
+# Default value is for launching from Meteo-France
+ARG INJECT_MF_CERT=1
```
[the remark this answers is not in the document]
Comment (A669015) [reply Dockerfile:8]: You do not need to default to 1, as you documented in the README to set it using the env var

**Review by A669015 — APPROVED** [review]

**Pull request merged**